src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEnginePublisher.java

```diff
@@ -315,15 +334,61 @@ public FormValidation doCheckZone(
       return FormValidation.ok();
     }
 
-    public FormValidation doCheckProjectId(@QueryParameter String value) {
-      if (Strings.isNullOrEmpty(value)) {
+    public ListBoxModel doFillProjectIdItems(
+        @AncestorInPath Jenkins context,
+        @QueryParameter("credentialsId") final String credentialsId) {
+      ListBoxModel items = new ListBoxModel();
+      items.add("");
```

Comment:
To ensure a consistent UI experience, suggest using the same empty label as zone and creds: "-none-"

Reply:
Done.